### DISCUSSION TITLE
Do blurred scrolled list heads and tails follow WinUI guidelines?

### DISCUSSION THREAD

**Discussion by xperiandri** [microsoft/microsoft-ui-xaml#7359] [Q&A] [answered]
We had such a conversation with @crutkas about the blurred list heads and tails.
See the full discussion here https://github.com/microsoft/PowerToys/issues/17435

![image](https://pbs.twimg.com/ext_tw_video_thumb/1454817561481199616/pu/img/4-UMEVjcrr0aY4wz.jpg)

Windows 11 in its UI ubiquitously uses ugly cropped scrolled list heads and tails contrary to the Microsoft To-Do app or the Microsoft Groove app.
So I wonder what are the official guidelines about that?
Is it really Microsoft wants ugly cropped list heads and tails, or it is just an inability of the developer or designer to make that beautiful?
@gabbybilka

**Comment by krschau** [comment]
@chigy might be able to weigh in on design guidance here.

**Reply by xperiandri** [reply]
@krschau, @chigy any updates?

**Comment by chigy** [comment, marked as answer]
@xperiandri , the guidance is not to show the blurry list under the header.

**Reply by xperiandri** [reply]
Thanks, sad
